### ISSUE TITLE
I am run it on server. whenever i have used npm run test-page

### ISSUE THREAD

**Issue by kul1010** [andyet/SimpleWebRTC#594]
Hi,
I am unable to run it on my server using below url
https://192.168.100.204:8443/test/
i am using this url on browser

![aaaaa](https://user-images.githubusercontent.com/13074125/27730444-e16f8640-5da6-11e7-8a91-3b176010b1cf.jpg)

**Comment by xdumaine** [comment]
@kul1010 It's only listening on localhost, not necessarily on that IP. A PR just merged to listed on all network interfaces, so could you try again?

**Comment by kul1010** [comment]
then tell me the branch name. So i can pull from that branch.

**Comment by xdumaine** [comment]
@kul I said it merged. It's in master.

**Comment by kul1010** [comment]
Also tell me, can i use my wss server for signalling rather than sandbox. like
url: 'wss://chat.asergis.org:7443/',

or
url: 'htpps://chat.asergis.org:7443/',

**Comment by xdumaine** [comment]
Yes you will need to so that. Use https.

**Comment by kul1010** [comment]
but not worked for link creation. show no url created like https://localhost:8443/test/?dfgdf
Also not worked for screen sharing.
Whenever wss://chat.asergis.org:7443 is opened.
see here
https://www.websocket.org/echo.html
![aaaaa](https://user-images.githubusercontent.com/13074125/27866435-ed4059f2-61b3-11e7-8021-b112f0798ac5.jpg)

**Comment by xdumaine** [comment]
Are you running [signalmaster](https://github.com/andyet/signalmaster)?

**Comment by kul1010** [comment]
I have run only this $ npm run test-page. And this wss://chat.asergis.org:7443 is running for freeswitch.
u can check old project with any name. but very slow and used sipml5. https://chat.asergis.org
But i want to use signalmaster .

**Comment by xdumaine** [comment]
You need to have an instance of signalmaster running as the websocket server, not just a generic web socket server. Please read through the documentation of both projects before continuing.

**Comment by kul1010** [comment]
ok sure. Thanks a lot!!!!!!
How to rebuild bundle for out/simplewebrtc-with-adapter.bundle.js

**Comment by fippo** [comment]
How about you read the documentation?
https://simplewebrtc.com/notsosimple.html#sandbox-server -- it is the first item on that page even

**Comment by kul1010** [comment]
I have changed in out\simplewebrtc.bundle.js  and given socket url as in config url: 'https://chat.asergis.org:3000/',

and run command as 
$ npm run build
$ npm run test-page

but not creating link to share url. 
![image](https://user-images.githubusercontent.com/13074125/27955638-62194876-6333-11e7-9d50-75be163aeaf7.png)

**Comment by kul1010** [comment]
What you have used at https://sandbox.simplewebrtc.com:443/
So create it Button is working means link creation functionality is working.
And i have given url in index.html as and taking this url when i create build again.
  var webrtc = new SimpleWebRTC({
                // the id/element dom element that will hold "our" video
                localVideoEl: 'localVideo',
                // the id/element dom element that will hold remote videos
                remoteVideosEl: '',
                // immediately ask for camera access
                autoRequestMedia: true,
                debug: false,
                detectSpeakingEvents: true,
                //  url: 'https://chat.asergis.org:3000'
                 url: 'https://sandbox.simplewebrtc.com:443/'
            });

and after it used $ npm run build 
$ npm run test-page

**Comment by kul1010** [comment]
Why localhost signalling server not working.
![image](https://user-images.githubusercontent.com/13074125/27959359-bfb42706-6344-11e7-833b-2ba909c2171e.png)

**Comment by xdumaine** [comment]
Is your signaling server running at that address?

**Comment by kul1010** [comment]
Yes. i run it after cloning from https://github.com/andyet/signalmaster.git  and install its dependencies using $ npm i
